lib/cocina/models.rb

```diff
@@ -153,5 +153,12 @@ def self.type_for(dyn)
       raise ValidationError, 'Type field not found'
     end
     private_class_method :type_for
+
+    def self.druid_regex
+      @druid_regex ||= begin
+        str = Openapi3Parser.load_file('openapi.yml').components.schemas['Druid'].pattern
```

Comment:
Are you sure the overhead of parsing openapi is worth it vs. just duplicating it here?

Reply:
Its memorized, so it should only be once per process. We can optimize it later if necessary.

Reply:
does openapi.yml strictly parse druids so the letters do not have vowels or the letter L ?

Reply:
https://github.com/sul-dlss/cocina-models/blob/main/openapi.yml#L1045